app/controllers/pages_controller.rb

```diff
@@ -126,20 +125,15 @@ def update_item
     item = @page.items.find_by id: item_params[:id]
     return unless item
 
-    item.assign_attributes item_params
-    item.save
+    item.update item_params
     item_page = ItemPage.find_by page_id: @page.id, item_id: item.id
     return unless item_page
 
     if params[:link_to_page].present?
       link_to_page = @spreadsheet.pages.find_by id: params[:link_to_page]
-      if link_to_page
-        item_page.link_to = link_to_page.name
-        item_page.save
-      end
-    elsif item_page.link_to.present?
-      item_page.link_to = nil
-      item_page.save
+      item_page.update(link_to: link_to_page.name) if link_to_page
+    elsif item_page.link_to?
+      item_page.update link_to: nil
```

Comment:
I'm not sure if I get the question here. Is it about the parentheses?

Reply:
Yep, which syntax should we follow

Reply:
I put parentheses in the method update line 134 because of if clause, let it more clear. But in the line 136, there is no ambiguity, I guess I didn't think much of it.
We can come with a convetion.

Reply:
I see, sounds good! No need to update